script/Arcueid/Day 9/09_00_ARC09_1.txt

```diff
@@ -147,7 +148,7 @@ I'm gazing down at the scene before me.
 [sha:5a3b9ab142cbd637aef5fa4d5995d6b55c2a071c]{
 -- Page 5, Offset 21321.
 -- 両手は倒れている子供と同じように赤い。
-My hands are red. Just like the child, collapsed in a heap on the floor.
+My hands are red, just like the child, collapsed in a heap on the floor.
```

Comment:
Alright, so originally I'd had this line pair as 
> My hands are red, just like the child now collapsed in a heap on the floor.
> My hands are red, with the blood of the living being now collapsed in a heap on the floor.

But it was suggested that maybe that's a bit wordy, and the commas were proposed. _However_, I feel that the comma in the second sentence of the pair is kinda easy to parse wrong, as though it is the hands that are collapsed:
> My hands are red with the blood of a living being, collapsed in a heap on the floor.

I don't really love the full stop, but it does at least separate out the sentence better. I might push for either 
1. going back to `now collapsed` with only one comma (after red)
2. breaking it with a full stop, but restating red to make it less abrupt, i.e.

> My hands are red. Red like the child collapsed in a heap on the floor.
> My hands are red. Red with the blood of the living being collapsed in a heap on the floor.

Reply:
Of those two proposals, I favor "My hands are red. Red like the child collapsed in a heap on the floor.". With that said, I don't think anybody is gonna read it as the _hands_ being collapsed. There's enough context that it's not an easy mistake.

Reply:
Yeah, I don't think anyone will leave the sentence with the wrong impression, it's just a bit of a hiccup.